src/Adornment/Adornment.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactNode } from 'react'
+
+import { AdornmentWrapper } from './styled'
+import { AdornmentContainer } from './AdornmentContainer'
+import { useAddWidthToPadding, useCloneChildWithPadding } from './hooks'
+
+export enum Position {
+  LEFT = 'Left',
+  RIGHT = 'Right'
+}
+
+export const ADORNMENT_DISPLAY_NAME = 'Adornment'
+export const EXTRA_PADDING = 'extraPadding'
```

Comment:
Any suggestions for this name? It is the padding object that gets passed to the child Adornment or Component

Reply:
perhaps inset?